quodlibet/browsers/playlists/main.py

```diff
@@ -145,6 +152,8 @@ def cell_data(self, col, cell, model, iter, data):
             return
         self.__last_render = markup
         cell.markup = markup
+        if isinstance(playlist, XSPFBackedPlaylist):
+            markup += " <small><tt>[XSPF]</tt></small>"
```

Comment:
Not a fan of this. Adds a lot of noise to the interface.

![Screenshot_20200908_203705](https://user-images.githubusercontent.com/3063858/92514965-1ba01d80-f213-11ea-9c5b-e9a1b4af53c6.png)

Reply:
Yeah, actually this was more for my debugging, I'll remove it. Thanks!